pkgs/top-level/python-packages.nix

```
@@ -1896,6 +1896,8 @@ in {
 
   emcee = callPackage ../development/python-modules/emcee { };
 
+  emv = disabledIf (pythonOlder "3.4") (callPackage ../development/python-modules/emv { });
```

please move inside expression
```suggestion
  emv = callPackage ../development/python-modules/emv { };
```

---

Done, thanks!